modules/benchmark/R/load.data.R

```diff
@@ -14,6 +14,8 @@ load.data <- function(data.path, format, start_year = NA, end_year = NA, site =
   
   library(PEcAn.utils)
   library(PEcAn.benchmark)
+  library(lubridate)
+  library(udunits2)
   library(dplyr)
```

Comment:
Per changes to style guide, package references should be explicit in the code (e.g. pkg::fcn), which means we don't need these library calls.  Specifically I believe we've successfully removed these three CRAN packages from everywhere else in PEcAn -- would be good to go ahead and clean up this code too.